config/authorization/config.ex

```diff
@@ -296,6 +301,20 @@ defmodule Acl.UserGroups.Config do
         ]
       },
 
+      %GroupSpec{
+        name: "admin-read",
+        useage: [:read],
+        access: access_by_role( admin_roles() ),
+        graphs: [
+          %GraphSpec{
+            graph: "http://mu.semte.ch/graphs/system/users",
```

Comment:
Als we toch voor de graph `http://mu.semte.ch/graphs/system/users` gaan (wat OK is voor mij aangezien we specifieke subclass `ext:LoginActivity` definieren), zou ik geen aparte group-spec maken en dit toevoegen aan de bestaande `admin` group-spec. Mu-auth laat dan ook schrijfrechten toe, wat mij niet problematisch lijkt voor admin gebruikers, maar dispatcher laat vanuit FE sowieso enkel `GET` requests door.

Note voor toekomst: als group-specs aangepast worden, vereist dit ook altijd aanpassing van de geconfigureerde eager indexing groups in mu-search config en cache-warmup.